Doc/library/typing.rst

```diff
@@ -500,6 +504,8 @@ The module defines the following classes, functions and decorators:
 
     A generic version of :class:`collections.abc.Reversible`.
 
+   .. versionadded:: 3.5.2
```

Comment:
In facts, there are 3 and 4 spaces indentation in this file.  What standard is expected ? I could make all the file standard